### ISSUE TITLE
Generated tests fail with psql timescale types

### ISSUE THREAD

**Issue by lukekhamilton** [volatiletech/sqlboiler#628]
Hey guys, 

I am in the process of testing out this package for use with a timescale/postgres table to see if these two things can play nicely together or not. So far I am hitting up with issue an issue when I run the tests. 

I have the following tables setup:

```
CREATE TABLE trade_data (
    time        TIMESTAMPTZ       NOT NULL,
    symbol      varchar(10)       not null,
    trade_id    integer           not null,
    price       double precision not null,
    quantity    integer not null,
    trade_type  integer null,
    PRIMARY KEY(time, symbol)
);

SELECT create_hypertable('trade_data', 'time', 'symbol', 100);
```

the `go generate ./...` works and creates the models without issue.

And this code also works:

```
db, err := sql.Open("postgres", "dbname=development user=postgres password=password sslmode=disable")
	if err != nil {
		log.Fatal(err)
	}

	boil.SetDB(db)

	td := models.TradeDatum{
		Time: time.Now(),
		Symbol: "BTCUSD",
		TradeID: 666,
		Price:600.0,
		Quantity:2000,
	}

	err = td.Insert(context.Background(), db, boil.Infer())
	if err != nil {
		log.Fatal(err)
	}

```

However I get the following errors when I run `go test -v ./...` I get a heap of test errors for all the different actions One, All, Count, Insert, etc...: 

```
--- FAIL: TestInsert (0.00s)
    --- FAIL: TestInsert/TradeData (0.01s)
        trade_data_test.go:457: models: unable to insert into trade_data: pq: invalid INSERT on the root table of hypertable "trade_data"
        trade_data_test.go:462: models: failed to count trade_data rows: pq: current transaction is aborted, commands ignored until end of transaction block
        trade_data_test.go:466: want one record, got: 0
    --- FAIL: TestInsert/TradeData#01 (0.01s)
        trade_data_test.go:484: models: unable to insert into trade_data: pq: invalid INSERT on the root table of hypertable "trade_data"
        trade_data_test.go:489: models: failed to count trade_data rows: pq: current transaction is aborted, commands ignored until end of transaction block
        trade_data_test.go:493: want one record, got: 0
=== RUN   TestToOne

```

Anyone got an idea why these tests are all failing but everything else is working? 

### What version of SQLBoiler are you using (`sqlboiler --version`)?

SQLBoiler v3.6.0

### What is your database and version (eg. Postgresql 10)

docker image: timescale/timescaledb:latest-pg11

**Comment by aarondl** [comment]
Typically the best way to debug this is to turn on log_statement in postgres and then run the specific test that's failing. That'll show us the actual sql being used in the test and can troubleshoot from there.

**Comment by troyanov** [comment]
That happens because during tests `pg_dump` is used to get DB Schema and then apply it to the temp test DB.

According to TimescaleDB docs:

> Restoring data from a backup currently requires some additional procedures, which need to be run from

https://docs.timescale.com/latest/using-timescaledb/backupdfg

I've tried naive approach by just adding these two lines:
```
SELECT timescaledb_pre_restore();
SELECT timescaledb_post_restore();
```
But it didn't work... 

The only way I got it working, was to use `pg_dump` and `pg_restore` but then faced another issue with FK. 

Maybe this information will be helpful.
